README.md

```diff
@@ -130,10 +139,12 @@ powershell "Start-Process -Wait -Verb RunAs powershell '-NoProfile iwr https://r
 3. Use one of these [GitHub Actions templates](templates/github-actions/README.md#frogbot-gitHub-actions-templates) to create a file named `frogbot.yml`.
 4. Push the `frogbot.yml` file to the `.github/workflows` directory in the root of your GitHub repository.
 
-### Install Frogbot Using GitLab CI
+### Setting up Frogbot on GitLab repositories
+Frogbot is installed on GitLab repositories using GitLab CI.
+Here's how you install it:
 
 1. Make sure you have the connection details of your JFrog environment.
-2. Save the JFrog connection details as secrets in GitLab with the following names: _JF_URL_, _JF_USER_ and _JF_PASSWORD_ (You can also use _JF_ACCESS_TOKEN_ instead of _JF_USER_ and _JF_PASSWORD_).
+2. Save the JFrog connection details as secrets in GitLab with the following names: JF_UR_, JF_USER and JF_PASSWORD (You can also use JF_ACCESS_TOKEN instead of JF_USER and JF_PASSWORD).
```

Comment:
JF_UR_ --> JF_URL

Also, the GitHub secrets above are bold. let's make the secrets of all platforms look the same.